src/sublime_aio.py

```diff
@@ -106,13 +132,13 @@ def _debounced_callback(view: sublime.View, coro_args: tuple) -> None:
             if call_at[view.view_id] <= __loop.time():
                 del call_at[view.view_id]
                 if view.is_valid():
-                    coro_func, self, args, kwargs = coro_args
-                    __loop.create_task(coro_func(self, *args, **kwargs))
+                    # Tasks are collected later using `asyncio.all_tasks`.
+                    _ = __loop.create_task(coro_func())
```

Comment:
Why assigning a nonused return value to just nothing? Isn't that just extra noise?

Reply:
Disabling all filters of basedpyright, I see the warning without `_ = `, but this is one of the runtime affecting messages, I find not worth it. Not only adding un-pythonic noise to the code it also executes useless assignment operation during runtime. Same as calling `cast()` shit.

Reply:
That was indeed a basedpyright warning, which has a couple functions' return values defined as "must use". The `_ =` syntax can be used to explicitly opt out of it.

I generally believe it makes sense to have this check, but it seemed a bit invasive on this code since there are quite a lot of instances with asynchronous computing where Python generally wants you to do something with the return value, mostly to cancel them.

I only added it in this particular case because it seemed important enough to even mention it in a comment. After switching to the `base` type checking mode, these warnings also don't show up anymore.